docs/examples/Power Flow Example.ipynb

```diff
@@ -663,6 +659,33 @@
     "update_data = {ComponentType.sym_load: update_sym_load, ComponentType.line: columnar_update_line}"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "f5179288",
+   "metadata": {},
+   "source": [
+    "Optional IDs for an update in columnar-data format are supported for uniform updates"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "id": "7eabed0c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "line_update_dtype = power_grid_meta_data[DatasetType.update][ComponentType.line].dtype\n",
+    "columnar_no_ID_update_line = {\n",
+    "    # Update IDs are not specified\n",
+    "    \"from_status\": np.array(\n",
+    "        [0, 1, 1], dtype=line_update_dtype[\"from_status\"]\n",
+    "    ),  # The update for the whole column needs to be specified\n",
```

Comment:
it's probably good to also show what happens if you don't do that (or explicitly mention that)